examples/default/.detoxrc.js

```diff
@@ -1,4 +1,69 @@
 /** @type {Detox.DetoxConfig} */
```

Comment:
Please move this comment right before the `module.exports = {` line since it's there as a type notation for it